app/Services/Registrar.php

```diff
@@ -85,7 +78,8 @@ public function create(array $data)
             }
 
             $user->country_code = get_country_code();
-            $user->northstar_id = $this->northstar->register($user);
+
+            $northstar_user = Northstar::createUser($user);
```

Comment:
I think we need to do some transformations before making this request. For example, Voting App saves phone numbers as `phone` whereas Northstar expects them to be given as `mobile`. (Although maybe we should just rename that database column to be consistent with what we use elsewhere?)

We should also attach a `source = <voting_app_name>` to this request so we can track where new Northstar users come from.